src/v2i-hub/TimPlugin/src/TimPlugin.cpp

```diff
@@ -0,0 +1,438 @@
+#include "TimPlugin.h"
+#include <WGS84Point.h>
+#include "TimeHelper.h"
+#include "XmlCurveParser.h"
```

Comment:
Some of there headers seem to be related to curve speed warning plugin and may not need to be used in TIM plugin, clean up.

Reply:
#include <WGS84Point.h> will be removed when we removed after we have successfully removed TestFindRegion. I will mark this one down for the web-service story.